src/guides/v2.4/release-notes/open-source-2-4-2.md

```diff
@@ -142,6 +142,12 @@ See the following articles for updates on features and changes for this release:
 
 *  [Yotpo Product Reviews]({{ site.baseurl }}/extensions/vendor/yotpo/release-notes.html)
 
+### AWS S3 support for the file storage
```

Comment:
Shouldn't this match the entry in the Commerce release notes?

```
### AWS S3 support enhancements

Amazon Simple Storage Service (AWS S3) support has been enhanced to include support for:

*  Object storage and future extensibility

*  [Storing media files]({{ page.baseurl }}/config-guide/remote-storage/config-remote-storage.html) on AWS S3

Support for AWS S3 has been added to all modules including B2B, PageBuilder, and Adobe Stock Integration.
```

Reply:
Interesting, I have not seen that.

Don't we need to mention it in OpenSource if that's available there?

Reply:
Yes, if it's available in both Open Source and Commerce, we need duplicate entries in both files.

Reply:
Thanks, then I will close my PR